### PR TITLE
*: run kubernetes tests in travis

### PR DESCRIPTION
**Pull request by ericchiang**
Just an experiment for now

### CODE REVIEW

**Comment by rithujohn191** [comment]
Do you want to add a note here https://github.com/coreos/dex/blob/master/Documentation/dev-integration-tests.md mentioning that we have added these tests to travis?

File: storage/kubernetes/storage.go [hunk 1]
```diff
@@ -164,6 +171,43 @@ func (cli *client) registerCustomResources(useTPR bool) (ok bool) {
...
+	err := cli.getResource("apiextensions.k8s.io/v1beta1", "", "customresourcedefinitions", name, &r)
+	if err != nil {
+		return fmt.Errorf("get crd %s: %v", name, err)
+	}
+	for _, c := range r.Status.Conditions {
+		if c.Type == k8sapi.Established && c.Status == k8sapi.ConditionTrue {
+			return nil
+		}
+	}
+	return fmt.Errorf("crd %s not ready", err)
```
Comment (ericchiang) [inline storage/kubernetes/storage.go]: woops, needs to be "name"

**Comment by ericchiang** [comment]
@rithujohn191 ready for review

File: scripts/test-k8s.sh [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    --etcd-servers=http://localhost:2379 \
+    --service-cluster-ip-range=10.0.0.1/16 \
+    --insecure-bind-address=0.0.0.0 \
+    --insecure-port=8080
+
+until $(curl --output /dev/null --silent --head --fail http://localhost:8080/healthz); do
+    printf '.'
+    sleep 1
+done
+echo "ok"
```
[the remark this answers is not in the document]
Comment (ericchiang) [reply scripts/test-k8s.sh]: Yeah, changed to "API server ready"

**Pull request merged**